packages/runtime/test/src/runtime.ts

```diff
@@ -205,6 +210,7 @@ describe("Algorand Standard Assets", function () {
     assert.equal(res.reserve, elon.address);
     assert.equal(res.freeze, elon.address);
     assert.equal(res.clawback, elon.address);
+    assert.equal(john.minBalance, initialMinBalance + ASSET_CREATION_FEE);
```

Comment:
here we are testing this case, in opcode-list this is repeated

Reply:
no, in opcode list we specifically test execution of `MinBalance` opcode. There is a slight difference. Here we directly test account's min balance. I would prefer to keep it.

Reply:
Ok, your call, better to have more tests